src/script/script.h

```diff
@@ -68,139 +68,148 @@ std::vector<unsigned char> ToByteVector(const T& in)
+    OP_NEGATE = 0x8f,    // multiply the top stack item by -1
+    OP_ABS = 0x90,       // absolute the top stack item value
+    OP_NOT = 0x91,       // convert 0 to 1 and else to 0
+    OP_0NOTEQUAL = 0x92, // return 0 if the input is 0, otherwise 1
+
+    OP_ADD = 0x93,    // pop top two stack items and push their sum
+    OP_SUB = 0x94,    // pop top two stack items and push the second minus the top
+    OP_MUL = 0x95,    // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_DIV = 0x96,    // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_MOD = 0x97,    // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_LSHIFT = 0x98, // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_RSHIFT = 0x99, // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+
+    OP_BOOLAND = 0x9a,            // return 1 if both top two stack items are not 0, otherwise 0
+    OP_BOOLOR = 0x9b,             // return 1 if top or second from top stack item is not 0, otherwise 0
+    OP_NUMEQUAL = 0x9c,           // return 1 if inputs are equal, otherwise 0
+    OP_NUMEQUALVERIFY = 0x9d,     // execute OP_NUMEQUAL, then OP_VERIFY afterward
+    OP_NUMNOTEQUAL = 0x9e,        // return 1 if inputs are not equal, otherwise 0
+    OP_LESSTHAN = 0x9f,           // pop top two stack items and push 1 if second from top < top, otherwise 0
+    OP_GREATERTHAN = 0xa0,        // pop top two stack items and push 1 if second from top > top, otherwise 0
+    OP_LESSTHANOREQUAL = 0xa1,    // pop top two stack items and push 1 if second from top <= top, otherwise 0
+    OP_GREATERTHANOREQUAL = 0xa2, // pop top two stack items and push 1 if second from top >= top, otherwise 0
+    OP_MIN = 0xa3,                // pop the top two items and push the smaller
+    OP_MAX = 0xa4,                // pop the top two items and push the bigger
+
+    OP_WITHIN = 0xa5, // pop the top three items and push 1 if third from top > top >= second from top, otherwise 0
 
     // crypto
-    OP_RIPEMD160 = 0xa6,
-    OP_SHA1 = 0xa7,
-    OP_SHA256 = 0xa8,
-    OP_HASH160 = 0xa9,
-    OP_HASH256 = 0xaa,
-    OP_CODESEPARATOR = 0xab,
-    OP_CHECKSIG = 0xac,
-    OP_CHECKSIGVERIFY = 0xad,
-    OP_CHECKMULTISIG = 0xae,
-    OP_CHECKMULTISIGVERIFY = 0xaf,
+    OP_RIPEMD160 = 0xa6,           // hash input using RIPEMD-160
+    OP_SHA1 = 0xa7,                // hash input using SHA-1
+    OP_SHA256 = 0xa8,              // hash input using SHA-256
+    OP_HASH160 = 0xa9,             // hash input using SHA-256, then RIPEMD-160
+    OP_HASH256 = 0xaa,             // hash input using SHA-256 twice
+    OP_CODESEPARATOR = 0xab,       // all of the signature checking words will only match signatures to the data after the most recently-executed OP_CODESEPARATOR
```

Comment:
I still have no idea how `OP_CODESEPARATOR` works, but that will take more than one code comment to explain.